composer.lock

```diff
@@ -199,16 +199,16 @@
         },
         {
             "name": "symfony/console",
-            "version": "v5.4.3",
+            "version": "v5.4.10",
```

Comment:
Update didn't change the version: needs work.

Reply:
Symfony v6 requires PHP >=8.0,2 https://packagist.org/packages/symfony/console#v6.0.0, and this will have run on PHP 7.4 (even though 7.3 is specified, but that's been fixed now).